plugin/src/node-builder.ts

```diff
@@ -109,33 +113,33 @@ const processorMap: ProcessorMap = {
 
 export function nodeBuilder(
   gatsbyApi: SourceNodesArgs,
-  options: ShopifyPluginOptions
+  pluginOptions: ShopifyPluginOptions
 ): NodeBuilder {
   return {
     async buildNode(result: BulkResult) {
       if (!pattern.test(result.id)) {
         throw new Error(
-          `Expected an ID in the format gid://shopify/<typename>/<id>`
+          `Expected an ID in the format gid://shopify/<typePrefix>/<id>`
```

Comment:
Woops, I don't think you meant to replace this!

Reply:
Dangit! My bad for not using case-sensitive search and replace. Good catch! Fixed!